practice-app/requirements.txt

```diff
@@ -12,3 +12,6 @@ requests==2.29.0
 urllib3==1.26.15
 Werkzeug==2.3.2
 zipp==3.15.0
+
+WTForms~=3.0.1
+SQLAlchemy~=2.0.12
```

Comment:
I think some packages are missing here. Can you also add these packages with their version: 
* flask_wtf
* flask_bootstrap
* flask_login
* psycopg2

Reply:
you can see the second commit on this pull request